mp/src/game/server/momentum/mom_player.cpp

```diff
@@ -1751,11 +1761,31 @@ void CMomentumPlayer::TimerCommand_Reset()
             // MOM_TODO do a trace downwards from the top of the trigger's center to touchable land, teleport the player there
             Teleport(&pCurrentZone->WorldSpaceCenter(), nullptr, &vec3_origin);
         }
-        else
+        return;
+    }
+
+    // Every other index is probably a stage (What about < 1 indexes? Mappers are weird and do "weirder"
+    // stuff so...)
+    CTriggerStage *pStage = nullptr;
+
+    while ((pStage = static_cast<CTriggerStage *>(gEntList.FindEntityByClassname(pStage, "trigger_momentum_timer_stage"))) != nullptr)
+    {
+        if (pStage && pStage->GetZoneNumber() == stage && pStage->GetTrackNumber() == track)
```

Comment:
`pStage` is nullchecked above and doesn't need null-checked again.
```suggestion
        if (pStage->GetZoneNumber() == stage && pStage->GetTrackNumber() == track)
```